.editorconfig

```diff
@@ -21,3 +21,7 @@ trim_trailing_whitespace = true
 indent_style = space
 indent_size = 4
 max_line_length = 80
+
+[{Makefile,_Makefile,**.mk}]
```

Comment:
In teh docs below you use only "Makefile", here also variants for including. Please be consistent.

Reply:
I removed the "**.mk" as it's indeed not needed. We need the _Makefile in this repository though, but normal repositories don't (because they have no example Makefile like us, that needs to protect indentation too)